test_scripts/API/VehicleData/GenericNetworkSignalData/027_VDI_permissions_in_next_ign_cycle.lua

```diff
@@ -51,6 +51,8 @@ local appSessionId = 1
 -- [[ Scenario ]]
 runner.Title("Preconditions")
 runner.Step("Clean environment", common.preconditions)
+runner.Step("Clear HMICapabilitiesCacheFile parameter in INI file", common.setSDLIniParameter,
+    { "HMICapabilitiesCacheFile", "" })
```

Comment:
```suggestion
  { "HMICapabilitiesCacheFile", "" })
```
I would propose to remove redundant whitespace here and in other scripts of PR